CONTRIBUTING.md

```diff
@@ -34,28 +39,29 @@ Below is a rough outline of building-secure-contracts's structure:
 
 ## Linters
 
-We run [markdown-link-check](https://github.com/tcort/markdown-link-check) to ensure all the markdown links are correct. 
+We run [markdown-link-check](https://github.com/tcort/markdown-link-check) to ensure all the markdown links are correct.
 
 To install `markdown-link-check`:
+
 ```bash
-$ npm install -g markdown-link-check
+npm install -g markdown-link-check
```

Comment:
> I really appreciate you taking the time to clean up so many formatting issues. I have a couple Qs but I like what I see. Since lots of changes are made in this PR already, we might want to get this batch merged before continuing to clean things up, merge conflicts will start accumulating quickly otherwise (mentioning this bc I see the PR is still marked as a Draft).
> 
> > made some preliminary checks to make sure nothing broke
> 
> What checks have you made so far? What other checks might we want to make?

I went through all the changes manually again and ran the server locally.

Reply:
when is this coming?